src/main/java/com/cloudbees/jenkins/support/impl/DirectoryComponent.java

```diff
@@ -19,7 +19,7 @@
 import java.util.logging.Logger;
 import java.util.stream.Stream;
 import jenkins.model.Jenkins;
-import org.apache.commons.lang.ArrayUtils;
+import org.apache.commons.lang3.ArrayUtils;
```

Comment:
This dep seems to be coming as a third-level transitive dep via `configuration-as-code`, which is `optional`, so this might break if that dep were disabled. Anyway it is pretty gratuitous.